crates/copilot_ui/src/copilot_button.rs

```diff
@@ -159,7 +159,11 @@ impl CopilotButton {
                 menu = menu.entry(
                     format!(
                         "{} Suggestions for {}",
-                        if language_enabled { "Hide" } else { "Show" },
```

Comment:
Is there a reason we're changing the terminology used here?

Reply:
I think the "Enable" / "Disable" and "Hide" is a better terminology. The current one, where "Disable" hides the copilot from status bar and "Hide from all files" disables it seems counter-intuitive... Happy to change it back if the team think something else is better.